tests/bettertransformer/test_bettertransformer_vision.py

```diff
@@ -66,3 +67,20 @@ def prepare_inputs_for_class(self, model_id=None):
         processor = AutoProcessor.from_pretrained(model_id)
         inputs = processor(image, text, return_tensors="pt")
         return inputs
+
+
+class BetterTransformersFlavaTest(BetterTransformersTestMixin, unittest.TestCase):
+    r"""
+    Testing suite for Vision and Text Models - tests all the tests defined in `BetterTransformersTestMixin`
+    """
+    all_models_to_test = ALL_VISION_TEXT_MODELS_TO_TEST
+
+    def prepare_inputs_for_class(self, model_id=None):
+        url = "http://images.cocodataset.org/val2017/000000039769.jpg"
+        image = Image.open(requests.get(url, stream=True).raw)
+        text = "How many cats are there?"
+
+        # Model takes image and text as input
+        processor = AutoProcessor.from_pretrained(model_id)
+        inputs = processor(image, text, return_tensors="pt")
+        return inputs
```

Comment:
```suggestion
        return inputs


     def  test_raise_activation_fun(self):
        r"""
        A tests that checks if the conversion raises an error if the model contains an activation function
        that is not supported by `BetterTransformer`. Here we need to loop over the config files
        """
        for hf_random_config in self.all_models_to_test:
            hf_random_config.vision_config.hidden_act = "silu"

            hf_random_model = AutoModel.from_config(hf_random_config).eval()
            with self.assertRaises(ValueError):
                _ = BetterTransformer.transform(hf_random_model, keep_original_model=True)      
```
Let's try to add this too, since this test is a bit trick ( you need to retrieve `config.vision_config`)

Reply:
Sounds good:) Again I really appreciate your help!